subworkflows/local/call_repeat_expansions/main.nf

```diff
@@ -54,19 +55,25 @@ workflow CALL_REPEAT_EXPANSIONS {
         .join( BCFTOOLS_SORT_TRGT.out.tbi )
         .map { meta, bcf, csi -> [ [ id : meta.family_id ], bcf, csi ] }
         .groupTuple()
-        .set{ ch_bcftools_merge_in }
+        .set{ ch_trgt_merge_in }
 
-    BCFTOOLS_MERGE (
-        ch_bcftools_merge_in,
-        ch_fasta,
-        ch_fai,
-        [[],[]]
+    TRGT_MERGE (
+        ch_trgt_merge_in,
+        [[],[]],
+        [[],[]],
     )
-    ch_versions = ch_versions.mix(BCFTOOLS_MERGE.out.versions)
+    ch_versions = ch_versions.mix(TRGT_MERGE.out.versions)
+
+    BCFTOOLS_INDEX(
+        TRGT_MERGE.out.vcf
+    )
```

Comment:
Need to collect version.

Reply:
Goshdarnit to heck 🙃
I always forget that

Reply:
We'll soon get topic channels and this will be a thing of the past ;)